openeo/extra/job_management.py

```diff
@@ -59,11 +59,32 @@ def read(self) -> pd.DataFrame:
     def persist(self, df: pd.DataFrame):
         """
         Store job data to the database.
+        The provided dataframe may contain partial information, which is merged into the larger database.
 
         :param df: job data to store.
         """
         ...
 
+    @abc.abstractmethod
+    def has_unfinished_jobs(self) -> bool:
```

Comment:
I don't think this has to be an exposed interface: it can just be done with `get_by_status` and the right status list. 
Note that `MultiBackendJobManager` is in charge of all the statuses it sets and expects, so for proper decoupling, a `JobDatabaseInterface` should not have to worry what statuses mean "unfinished"

Reply:
yes, although a job database could optimize this implementation rather than having to retrieve the full list of unfinished jobs?
We could also go for something like 'has_jobs_with_status'
or 'count_by_status'?

Reply:
Proposed: count_by_status